src/Opdex.Platform.WebApi/Controllers/TokensController.cs

```diff
@@ -97,33 +98,23 @@ public async Task<ActionResult<TokenResponseModel>> GetToken([FromRoute] Address
         }
 
         ///<summary>Get Token History</summary>
-        /// <remarks>Retrieve historical data points for a tokens tracking open, high, low, and close of USD prices.</remarks>
+        /// <remarks>Retrieve historical data points for a token tracking open, high, low, and close of USD prices.</remarks>
         /// <param name="address">The address of the token.</param>
-        /// <param name="candleSpan">"Hourly" or "Daily" determining the time span of each data point. Default is daily.</param>
-        /// <param name="timespan">"1D", "1W", "1M", "1Y" determining how much history to fetch. Default is 1 week.</param>
-        /// <param name="cancellationToken">Cancellation token</param>
-        /// <returns><see cref="TokenSnapshotHistoryResponseModel"/> with a list of historical data points.</returns>
+        /// <param name="filters">Filter parameters.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <returns>Paged token snapshot data.</returns>
         [HttpGet("{address}/history")]
-        [ProducesResponseType(typeof(TokenSnapshotHistoryResponseModel), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(TokenSnapshotsResponseModel), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
```

Comment:
For future though what do you think about not returning 404's on filter related requests? Alternatively, returning empty result set instead. So when market || token are not found, just exit early instead. 

Wouldn't be for this task, I think all of our filter endpoints are operating this way with 404's but might be worth ignoring and returning empty results.

Reply:
Disagree with this, the 404 status code is the right way to handle it. Guessing you're suggesting this in relation to exception logs, in which case I think we shouldn't use, or should suppress exceptions so they're not logged.